docs/network_firewall.rst

```diff
@@ -465,20 +469,37 @@ to add a rule.
 Once you've set up the firewall, exit the Unsafe Browser, and continue
 with the "Keeping pfSense up to date" section below.
 
-Configuration Reference Templates
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+SecureDrop Configuration (Templates)
+------------------------------------
+
+As an alternative to the manually provided screenshots, you can use our ``.xml``
+templates. However, because of the Tails chroot, you can not download the
```

Comment:
Is it explained what the `Tails chroot` is? We should briefly explain (without assuming the reader is highly technical :wink:).

Reply:
Agreed, let's elide "chroot" altogether. We can simply provide instructions to use the Unsafe Browser. It may be reasonable to provide a `curl` command (coupled with `torify` if necessary).